onward/app/views/feedback.scala.html

```diff
@@ -4,56 +4,65 @@
-                                        <p>If your problem includes the following</p>
-                                        <ul>
-                                            <li>crashing</li>
-                                            <li>reloading</li>
-                                            <li>very slow site access</li>
-                                        </ul>
-                                        <p>please learn about <a href="@LinkTo{/info/2015/sep/22/making-theguardiancom-work-best-for-you}">disabling optional features on our site</a></p>
-                                        <h2>Accessibility</h2>
-                                        <p>If you have an issue related to the accessibility of this website, please see our <a href="https://www.theguardian.com/help/accessibility-help">accessibility help page</a>.</p>
-                                        <h2>Contact us</h2>
-                                        <p>If you need help, please contact our user support desk at
-                                            @* Any link with mailto userhelp will be enhanced using the Javascript *@
-                                            <a data-link-name="tech feedback : mailto : userhelp" href="mailto:userhelp@@theguardian.com">
-                                                userhelp@@theguardian.com
-                                            </a> and someone should get back to you.
-                                        </p>
-                                        <p>
-                                            If you don't require any assistance but wish to report an issue to our developers, send comments to:
-                                            <a class="js-tech-feedback-mailto" data-link-name="tech feedback : mailto : dotcom" href="mailto:dotcom.feedback@@theguardian.com">
-                                                dotcom.feedback@@theguardian.com
-                                            </a>.<br>
-                                            <strong>We value and read all of the feedback that we receive, but we are unable to respond directly.</strong>
-                                        </p>
-                                        <p>Think you can fix it yourself? <a href="http://developers.theguardian.com/">Work for us</a>!</p>
+                </div>
+            </header>
+            <div class="content__main">
+                <div class="gs-container">
+                    <div class="content__main-column content__main-column--expired">
+                        <div class="js-article__container u-cf">
+                            <div class="from-content-api" itemprop="articleBody">
+                                <div class="from-content-api">
+
+                                    <h2>Contact us</h2>
+
+                                    <div class="feedback__element">
+                                        <label class="" for="test">Please choose a service below </label>
```

Comment:
Kill the `class` attribute if we don't need it. Also the `for` attribute doesn't appear to be pointing at anything? Perhaps it ought to be `for="feedback-category"`?